build/helper/documentation_snippets.py

```diff
@@ -8,10 +8,10 @@
 '''
 
 rep_cap_attr_desc = '''
-This property can use repeated capabilities for {0}. If set or get directly on the
-{1}.Session object, then the set/get will use all repeated capabilities in the session.
-You can specify a subset of repeated capabilities using the Python index notation on an
-{1}.Session repeated capabilities container, and calling set/get value on the result.
+This property can be set/get to specific {1} within your {0}.Session object. Use Python index notation on the repeated capabilities container `{2}` to specify a subset.
```

Comment:
```suggestion
This property can be set/get on specific {1} within your {0}.Session object. Use Python index notation on the repeated capabilities container `{2}` to specify a subset.
```

Reply:
Can you also please break up the line to be within the the limit we had before.

Reply:
Ok, I'll make these edits in my local branch, alongside the line break changes.

Reply:
@luisgomes252, I don't see this change in the latest commit.

Reply:
Are you looking at the diff in the **Conversation** tab? The comment shows up with an "Outdated" tag, which I assume means that the associated diff is no longer relevant because a subsequent commit has modified it.

When I look at the overall diff in the **Files Changed** tab, I see the latest committed changes, which looks correct to me:

![image](https://user-images.githubusercontent.com/80292064/119552561-4827a580-bd60-11eb-8a34-337783538be8.png)

Reply:
I have discussed this offline with @sbethur and he has clarified that he was referring to some of the wording changes he had suggested ("object" -> "instance", "to" -> "on"), which I hadn't noticed before. I've now made those changes.